samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,100 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
+##################
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
+
+Overview
+********
+
+The sample allows to build Thread Border Router target on the MCU. 
+It is important to mention that, according to NCP architecture, MCU part
+needs to cooperate with user higher layer process to altogether establish complete 
+Thread Border Router. Recommended solution is to use docker version of OTBR (OpenThread Border Router)
+and WPAN Network Daemon tool (wpantund), working under Linux operating system, for such purpose 
+(for more information read this: https://openthread.io/guides/border-router).
+
+There is possibility to configure number of OpenThread features, but by default
+application has COMMISSIONER, JOINER, BORDER_ROUTER, BORDER_AGENT and DHCP6_SERVER
+functionalities enabled.
+
+Requirements
+************
+
+* One of the following development kits for testing the network status:
+
+  * |nRF52840DK|
+  * |nRF52833DK|
+
+* Border Router higher layer process communicating with MCU border router part.
+
+* Optionally, one or more compatible development kits programmed with OpenThread samples to test communication
+  in bigger network.
+
+User interface
+**************
+
+Main recommended interaction with the application should be done via wpantund
+Linux process using its comannds (for more information: https://github.com/openthread/wpantund).
```

Comment:
don't link directly. Add the link into proper section of https://github.com/nrfconnect/sdk-nrf/blob/master/doc/nrf/links.txt and ref it here.
Doc coding standard: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#hyperlinks

Reply:
Done